tests/test_api.py

```
@@ -0,0 +1,320 @@
+    """Test complete integration flow."""
+
+    def test_full_anomaly_detection_flow(self, client):
+        """Test complete flow: submit telemetry -> detect anomaly -> check history."""
+        # 1. Check initial status
+        status_response = client.get("/api/v1/status")
+        assert status_response.status_code == 200
+
+        # 2. Get current phase
+        phase_response = client.get("/api/v1/phase")
+        assert phase_response.status_code == 200
+        initial_phase = phase_response.json()["phase"]
```

Ruff (0.14.10)

Local variable `initial_phase` is assigned to but never used

Remove assignment to unused variable `initial_phase`

(F841)